locale/pt-br/texts.po

```diff
@@ -57,7 +57,7 @@ msgid "[Last] dddd [•] HH:mm"
 msgstr "[Última] dddd [•] HH:mm"
 
 #.  See https://momentjs.com/docs/#/displaying/calendar-time/
-#: src/models.js:107 src/utils.js:426
+#: src/models.js:107 src/utils.js:427
```

Comment:
Why this line changed in `src/utils`?

Reply:
I ran an update, probably it was lacking an update.